client/src/origo.rs

```diff
@@ -38,12 +38,13 @@ pub struct VerifyBody {
   pub manifest:    OrigoManifest,
 }
 
-// TODO: Okay, right now we just want to take what's in here and actually just produce a signature
-// as the reply instead. So pretend this is signed content for now and not actual raw values.
-#[derive(Serialize, Deserialize, Debug, Clone)]
-pub struct VerifyReply {
-  pub value:    String,
-  pub manifest: Manifest,
+#[derive(Debug, Deserialize, Serialize, Clone)]
+pub struct OrigoProof {
+  pub proof:             FoldingProof<Vec<u8>, String>,
+  pub rom:               NIVCRom,
+  pub ciphertext_digest: [u8; 32],
+  pub sign_reply:        Option<SignedVerificationReply>,
+  pub value:             Option<String>,
```

Comment:
q: I'm assuming `value` is a hex-encoded string of bytes made from a notary response. Is it okay to put _whatever_ in there (e.g. a JSON-object)?

Reply:
yeah, we should have support for these in the circuits eventually, maybe a different change for origo